src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -340,6 +341,26 @@ trait DeploymentFormats {
   }
 
   implicit lazy val DeploymentStepWrites: Writes[DeploymentStep] = Json.writes[DeploymentStep]
+
+  implicit lazy val DeploymentStepInfoWrites: Writes[DeploymentStepInfo] = Writes { info =>
+    def deploymentAction(action: DeploymentAction): JsObject =
```

Comment:
I think this isn't necessary, `DeploymentActionWrites` already exists and does exactly the same.